datafusion/expr/src/logical_plan/plan.rs

```diff
@@ -1625,6 +1637,15 @@ pub struct Prepare {
     pub input: Arc<LogicalPlan>,
 }
 
+/// Describe a file
```

Comment:
What would you think about making this more general `DescribeTable` and rather than taking a file_path it would have a `table_provider`? I think then we wouldn't have to special case describe for file / table -- instead DataFusion could always create `DescribeTable` 

The query in terms of information_schema is sort of a hack in the first place. What you have started here with an actual LogicalPlan variant I think is much better 👍

Reply:
done